Kitodo-DataFormat/src/main/java/org/kitodo/dataformat/access/DivXmlElementAccess.java

```diff
@@ -216,8 +229,8 @@ DivType toDiv(Map<MediaUnit, String> mediaUnitIDs, LinkedList<Pair<String, Strin
             mets.getAmdSec().add(admSec);
         }
 
-        for (Structure substructure : super.getChildren()) {
-            div.getDiv().add(new DivXmlElementAccess(substructure).toDiv(mediaUnitIDs, smLinkData, mets));
+        for (ExistingOrLinkedStructure substructure : super.getChildren()) {
+            div.getDiv().add(new DivXmlElementAccess((Structure) substructure).toDiv(mediaUnitIDs, smLinkData, mets));
```

Comment:
The design is currently being changed. In this pull request, I can assume at this point that I only get structures, because this is a method to save, but this pull request introduces only the methods for reading. This location is then [changed in the pull request for saving linked structures](https://github.com/kitodo/kitodo-production/pull/2150/files#diff-312f5490b49dba85a606dfbedf437e4cR220). So I'm assuming at this point that you do not yet save any linked structures, but only with the code from the second pull request, which then can also.